<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>B_Tree</class>
 <widget class="QWidget" name="B_Tree">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1049</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LES ARBRES </string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="le_data">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Saisir une valeur</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="ArbreTypes">
       <property name="duplicatesEnabled">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>ABR</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>AVL</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>TAS min</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>TAS max</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>B-arbre</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>M-aire</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="IF">
       <property name="text">
        <string> cr√©ation depuis un fichier</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="SR">
       <property name="text">
        <string>supprimer la racine</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="OF">
       <property name="text">
        <string>Ovrire le fichier</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>22</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>Order</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="grade_sb">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>44</width>
         <height>25</height>
        </size>
       </property>
       <property name="keyboardTracking">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <number>3</number>
       </property>
       <property name="maximum">
        <number>10</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>3</number>
       </property>
       <property name="displayIntegerBase">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="find">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bt_add">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>img/img_add.png</normaloff>img/img_add.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_remove">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>img/img_sub.png</normaloff>img/img_sub.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_clear">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Operations</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QScrollArea" name="scrollArea">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>860</width>
          <height>424</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="tree_lb">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QListWidget" name="lw_operations">
       <property name="maximumSize">
        <size>
         <width>161</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
